src/OpenTelemetry.Instrumentation.AWS/Implementation/Utils.cs

```diff
@@ -22,17 +22,12 @@ internal class Utils
 
     internal static string RemoveSuffix(string originalString, string suffix)
     {
-        if (string.IsNullOrEmpty(originalString))
-        {
-            return string.Empty;
-        }
-
-        if (originalString.EndsWith(suffix, StringComparison.Ordinal))
-        {
-            return originalString.Substring(0, originalString.Length - suffix.Length);
-        }
-
-        return originalString;
+        return string.IsNullOrEmpty(originalString)
```

Comment:
nit: Do we have to make this change? The previous version was more readable to me.